src/main/build/debug_pin.h

```diff
@@ -20,6 +20,22 @@
 
 #pragma once
 
+#include <drivers/io_types.h>
+
+typedef struct dbgPin_s {
+    ioTag_t tag;
+} dbgPin_t;
+
 void dbgPinInit(void);
 void dbgPinHi(int index);
 void dbgPinLo(int index);
+
+#ifdef USE_DEBUG_PIN
+#define DEBUG_LO(index) \
```

Comment:
What about the backslash?

Reply:
It's a continuation, the so that the definition is on the following line, quite common in the codebase.